docs/aca/03-aca-dapr-integration/index.md

```diff
@@ -92,103 +102,116 @@ When we complete the previous step, we are ready to run the applications locally
+
+    ~\TasksTracker.ContainerApps\TasksTracker.WebPortal.Frontend.Ui> dapr run --app-id tasksmanager-frontend-webapp --app-port [web frontend application port found under properties->launchSettings.json. e.g. 7000] --dapr-http-port 3501 --app-ssl -- dotnet run 
+
+    ~\TasksTracker.ContainerApps\TasksTracker.TasksManager.Backend.Api> dapr run --app-id tasksmanager-backend-api --app-port [web api application port found under properties->launchSettings.json. e.g. 7112] --dapr-http-port 3500 --app-ssl -- dotnet run
+
+    If you are on .Net 7 or above use the following commands:
+
+    ~\TasksTracker.ContainerApps\TasksTracker.WebPortal.Frontend.Ui> dapr run --app-id tasksmanager-frontend-webapp --app-port [web frontend application port found under properties->launchSettings.json. e.g. 7000] --dapr-http-port 3501 --app-ssl -- dotnet run --launch-profile https
 
-    ~\TasksTracker.ContainerApps\TasksTracker.TasksManager.Backend.Api> dapr run --app-id tasksmanager-backend-api --app-port 7088 --dapr-http-port 3500 --app-ssl dotnet run
+    ~\TasksTracker.ContainerApps\TasksTracker.TasksManager.Backend.Api> dapr run --app-id tasksmanager-backend-api --app-port [web api application port found under properties->launchSettings.json. e.g. 7112] --dapr-http-port 3500 --app-ssl -- dotnet run --launch-profile https
     ```
 
     Notice how we assigned the Dapr App Id “tasksmanager-frontend-webapp” to the Frontend WebApp.
 
-    Now both Applications are running using Dapr sidecar, open your browser and browser for "https://localhost:{localwebappport}", in my case it will be "https://localhost:7208" and provide an email to load the tasks for the user, if all is good you should see tasks list results.
+    Now both Applications are running using Dapr sidecar. Open your browser and browse for "https://localhost:{localwebappport}". E.g. "https://localhost:7000" and provide an email to load the tasks for the user (e.g. tjoudeh@bitoftech.net). If the application is working as expected you should see tasks list associated with the email you provided (e.g. tjoudeh@bitoftech.net).
 
-**Note:** If we need to run both microservices together, we need to keep calling `dapr run` manually each time in the terminal, and when we have multiple microservices talking to each other and need to run at the same time to debug the solution, this will be a very annoying process. You can refer to the [debug and launch Dapr applications in VSCode](xxx) to see how configure VScode for running and debugging Dapr applications.
+**Note:** If you need to run both microservices together, you need to keep calling `dapr run` manually each time in the terminal. And when you have multiple microservices talking to each other you need to run at the same time to debug the solution. This can be a convoluted process. You can refer to the [debug and launch Dapr applications in VSCode](xxx) to see how to configure VScode for running and debugging Dapr applications.
```

Comment:
Please use this link and replace xxx with it:
../../aca/20-appendix/01-run-debug-dapr-app-vscode.md